install/linux/docker-ce/ubuntu.md

```diff
@@ -138,7 +138,8 @@ the repository.
         apt-transport-https \
         ca-certificates \
         curl \
-        software-properties-common
+        software-properties-common \
```

Comment:
Could you keep this list sorted alphabetically?

```suggestion
        gpg-agent \
        software-properties-common
```

Reply:
Fixed.